modules/Data_Cleaning/Data_Cleaning.Rmd

```diff
@@ -50,7 +50,7 @@ In general, data cleaning is a process of investigating your data for
 inaccuracies, or recoding it in a way that makes it more manageable.
 
 ::: {style="color: red;"}
-⚠️ MOST IMPORTANT RULE - LOOK 👀 AT YOUR DATA! ⚠️
+`r emo::ji("warning")` MOST IMPORTANT RULE - LOOK `r emo::ji("eyes")` AT YOUR DATA! `r emo::ji("warning")`
```

Comment:
@carriewright11 I did this because the ⚠️ was triggering a spelling error. Not sure why the other emojis are not.

Reply:
yeah I noticed that :( I thought about trying to add it to the dictionary but didn't get a chance

Reply:
I figured you couldn't add emojis to the dictionary 😜